ui/desktop/app/styles/app.scss

```diff
@@ -291,3 +290,64 @@
   margin: sizing.rems(l) 0 0 0;
   text-align: center;
 }
+
+
+// Dialog Detail
+// These styles are a Work In Progress. Expect changes (lots of them)
+.connection-dialog {
+  .rose-notification {
+    border-left: 0;
+    border-right: 0;
+    border-top: 0;
+    box-shadow: none;
+    margin-bottom: 0;
+  }
+
+  .rose-dialog-footer {
+    align-items: center;
+    display: flex;
+    
+    div:nth-child(1) {
+      display: flex;
+      justify-content: flex-start;
+      width: 50%;
+    }
+      
+    div:nth-child(2) {
+      display: flex;
+      justify-content: flex-end;
+      width: 50%;
+    }
+  }
+
+  // .credentials-card {
```

Comment:
Please, ignore this commented code. I am using it for reference and I will get rid of it pretty soon.